mne/viz/evoked.py

```diff
@@ -162,6 +199,16 @@ def _plot_evoked(evoked, picks, exclude, unit, show,
                     ax._get_lines.color_cycle = iter(colors)
                 else:
                     ax._get_lines.color_cycle = cycle(['k'])
+
+                callback_onselect = partial(_butterfly_onselect,
+                                            ch_types=ch_types_used,
+                                            evoked=evoked)
+                blit = False if plt.get_backend() == 'MacOSX' else True
```

Comment:
In cases like this I often do `...ackend().lower() == 'macosx'` as an alternative to getting the caps right. Not sure which is better, now that I think about it...